vuepress/docs/v6.3.2/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -42,100 +46,117 @@ customUi: <h2>Hi from Example Widget</h2>
 components:
   widgets:
     - widgets/example-widget.yaml
 ```
-The component descriptor file name and location (e.g. `widgets/example-widget.yaml`) are arbitrary since the bundle descriptor explicitly points to the file. The typical convention is to group components by type, for example with all widgets in one directory, all page templates in another, etc.
+Component descriptor file names and locations (e.g. `widgets/example-widget.yaml`) are arbitrary since the bundle descriptor explicitly points to those files. Convention is to group components by type (e.g. all widgets in one directory, all page templates in another, etc.).
 
 ## Publish the bundle
 
-You can publish a bundle using the CLI or you can perform the steps by hand.
+The bundle can be published using the `ent prj` command, or the equivalent steps can be performed manually.
 
-### CLI steps
-1. Change to the project directory, if needed
+### CLI steps with `ent prj`
+1. Change to the project directory if needed
 ```sh
 cd example-bundle
 ```
 
-2. Initialize the Entando project and accept the defaults.
+2. Initialize the Entando project and accept the defaults
 ``` sh
 ent prj init
 ```
-3. Initialize the publication system. You'll need the git repository URL and your credentials here.
+
+3. Initialize the publication system. This step requires the empty Git repository URL (ending in .git) and your credentials.
 ``` sh
 ent prj pbs-init
 ```
-4. Publish the bundle to git. By convention your first version will be `v0.0.1` but this is up to you.
+
+4. Publish the bundle to Git. By convention the first version is assigned the tag `v0.0.1`. This tag can be modified to use your preferred versioning format.
 ``` sh
 ent prj pbs-publish
 ```
-On subsequent iterations you can run just this command again to quickly push a new version of your bundle to git.
+Running just the command `ent prj pbs-publish` will quickly push subsequent iterations of the bundle to Git. You will be asked to input the bundle version each time. To ensure that iterations are listed in the correct order you must be consistent with versioning format and alphanumeric precedence.
 
-5. You can now deploy the bundle into the Entando Component Repository in one step.
+5. The bundle can now be deployed into the Entando Component Repository with one command
 ``` sh
 ent prj deploy
 ```
- The `prj deploy` command uses the git repository URL and project name from earlier steps to create the custom resource.
+ The `prj deploy` command uses the Git repository URL and project name (e.g. `example-bundle`) to create the custom resource.
 
-6. Jump to the section below to finish installing your bundle: [Install the bundle into an application](#install-the-bundle-into-an-application)
+6. Jump to [Install the bundle into an application](#install-the-bundle-into-an-application) to finish installing your bundle.
 
-### Manual steps
-1. Change to the bundle directory, if needed
+### Unsimplified CLI steps 
```

Comment:
Let's discuss the Manual vs Unsimplified. We've added CLI vs Manual sections in quite a few tutorials and Unsimplified doesn't feel quite right to me. Typically a CLI automates steps and the opposite of automated would be manual in that context.

Reply:
thank you! a fundamental misconception strikes again! "ent" = Entando CLI clears it up on my end - i'll revert to CLI and manual for headers and descriptions. i struggled for awhile with how to differentiate what i was interpreting as more or less commands from the terminal, equating the terminal with command line interface... unsimplified and condensed/consolidated read very awkwardly and i wasn't happy with the terminology so uploaded as a draft :P